docs/bot/setup_debian.md

```diff
@@ -0,0 +1,137 @@
+# Installation on Debian
+These are instructions for the installation of draupnir from source on Debian  
+This installation method is intended for experienced sysadmins.
+
+## Installation
+install git curl and sudo
+```shell
+apt update && apt install -y git curl sudo
+```
+install node 18 from the node source repo, the full instructions can be found at https://github.com/nodesource/distributions
+```shell
+curl -fsSL https://deb.nodesource.com/setup_18.x -o nodesource_setup.sh
+bash nodesource_setup.sh
+apt update && apt install nodejs -y
+```
+install yarn via npm
+```shell
+npm install --global yarn                                                                                                                    # install yarn via npn
+```
+create the directory to clone the repo
+```shell
+mkdir /opt/mod-bot  
+```
+clone the repo
+```shell
+git clone https://github.com/the-draupnir-project/Draupnir.git /opt/mod-bot/Draupnir
+```
+create the directory for draupnirs datastorage
```

Comment:
Because Github is dumb or i cant format this correctly il just type what i suggest should exist in some form between L27 and L28.

>Fetch tags to make draupnir happy
>```shell
>bash -c "cd /opt/mod-bot/Draupnir && git fetch --tags"
>```


Im not sure this is the exact way to do this but like we have to fetch the tags or else Draupnir will be unhappy as far as i understand.

Reply:
added the command to fetch the tags :) 

```shell
git -C /opt/mod-bot/Draupnir fetch --tags
```